### PR TITLE
Feature/ee2 final wafs

### PR DESCRIPTION
**Pull request by SamiraArdani-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>
in this PR, the evs_ver was modified. In addition, all the errfiles were concatenated into stdout. Scripts were tested.

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
There is no prep STEP for wafs component.
To run for STEP stats: /dev/drivers/scripts/stats/wafs/jevs_wafs_atmos_stats.sh 
To run for STEP plots:/dev/drivers/scripts/stats/wafs/jevs_wafs_atmos_plots.sh

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

File: scripts/plots/wafs/exevs_wafs_atmos_plots.sh [hunk 1]
```diff
@@ -119,10 +119,25 @@ for RESOLUTION in $resolutions ; do
...
+		echo "Start: $log_file"
+		cat $log_file                                                                   
+		echo "End: $log_file"
+	done
+fi
+
+
 #####################################################################
 # GOOD RUN
 echo "********SCRIPT exevs_wafs_atmos_plots.sh $1 $2 COMPLETED NORMALLY on `date`"
```
Comment (ShelleyMelchior-NOAA) [inline scripts/plots/wafs/exevs_wafs_atmos_plots.sh:139]: @SamiraArdani-NOAA , NCO doesn't want us to use the unix date command. Can you try `$NDATE` in line 139 in place of date? $NDATE is a NCO utility

**Comment by AliciaBentley-NOAA** [comment]
@SamiraArdani-NOAA Thank you so much for working on the Final EE2 PR for WAFS. I have a few comments for you:

1) Can you please remove the commented out lines in scripts/stats/wafs/exevs_wafs_atmos_stats.sh?
       Line 72:     # COMOUTsmall

2) Can you please remove the commented out lines in scripts/plots/wafs/exevs_wafs_atmos_plots.sh?
	Line 61:    #rm $OUTPUT_BASE_DIR/*
	Line 87:    #rm $OUTPUT_BASE_DIR/*

3) Should there be "`export err=$?; err_chk`" somewhere in scripts/plots/wafs/exevs_wafs_atmos_plots.sh? Seems like there should be, right? 

4) It looks like [Line 90 in scripts/plots/wafs/exevs_wafs_atmos_plots.sh](https://github.com/NOAA-EMC/EVS/pull/333/files#diff-2ceb465e729cb837ba0edcdd8c50d7c9d0f6e23d0b7314637a4cd8271494caa3R90) uses "date" instead of PDY or NDATE. Please see @ShelleyMelchior-NOAA's separate suggestion to use NDATE. 

That's all for now! Thanks again for doing this!

**Comment by AliciaBentley-NOAA** [comment]
@SamiraArdani-NOAA One quick clarifying question: 

Did anything in the [WAFS /ush scripts](https://github.com/NOAA-EMC/EVS/tree/develop/ush/wafs) need to change for this Final EE2 PR? Specifically in the "stats" ush scripts?

**Comment by ShelleyMelchior-NOAA** [comment]
Did we have a final EE2 mtg for WAFS? I think @SamiraArdani-NOAA only covered priorities 4 and 5 in this PR (for example, why unix date was not changed (yet)). But also, since plots is not being delivered, we can defer plots processing changes in the interest of time. What do you think, @AliciaBentley-NOAA ?

**Comment by PerryShafran-NOAA** [comment]
@ShelleyMelchior-NOAA @AliciaBentley-NOAA agreed; let's not worry about fixing plot files for now.  I'd suggest sticking to stats changes.

**Comment by AliciaBentley-NOAA** [comment]
@ShelleyMelchior-NOAA @PerryShafran-NOAA Correct. We did not have a Final EE2 checklist meeting for WAFS. I thought that those topics were covered in this PR because of it's name ("ee2 final wafs"). @SamiraArdani-NOAA Are "Final EE2 PR" topics not covered in this "ee2 final wafs" PR?

I agree about just focusing on WAFS stats (dev drivers, j-jobs, scripts, ush scripts, and parm files) in the interest of time.

**Comment by PerryShafran-NOAA** [comment]
@SamiraArdani-NOAA - I am testing this PR now.  I am only focusing on the stats since plots is not going to be an operational product.

**Comment by SamiraArdani-NOAA** [comment]
@AliciaBentley-NOAA : we have covered most of the EE2 items in wafs. I just saw those unix "date" commands and one syntax  "no such file or directory" in one of the .o files. we did not have jlogfile, postmsg, etc in wafs but I double check and let you know in the meeting.

**Comment by AliciaBentley-NOAA** [comment]
@PerryShafran-NOAA Good morning! Is there WAFS stats test output from yesterday to view?

**Comment by PerryShafran-NOAA** [comment]
@AliciaBentley-NOAA @SamiraArdani-NOAA 

Yes, we have output to review!

Please look at the .o file here:  /lfs/h2/emc/vpppg/noscrub/perry.shafran/pr333test/EVS/dev/drivers/scripts/stats/wafs/jevs_wafs_stats.o107978152

And the output stats are here:

/lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/stats/wafs/____.20231101

**Comment by SamiraArdani-NOAA** [comment]
@PerryShafran-NOAA : I will review and get back to you soon.

**Comment by SamiraArdani-NOAA** [comment]
@PerryShafran-NOAA : The outputs were checked with auth repository and they are identical.

**Comment by PerryShafran-NOAA** [comment]
Good news!  Please do a sync fork and then we will merge.

**Comment by SamiraArdani-NOAA** [comment]
@PerryShafran-NOAA : I did a sync fork. 
@AliciaBentley-NOAA @PerryShafran-NOAA : I just wanted to confirm: Do we have another PR after our meeting this afternoon? or you want to keep this PR and I commit the rest of changes here in this PR and then you merge?

**Comment by PerryShafran-NOAA** [comment]
@SamiraArdani-NOAA We finalize and merge this PR.  Your final EE2 changes come in a new PR.

**Comment by AliciaBentley-NOAA** [comment]
@SamiraArdani-NOAA We can do another PR after this one. I'd like to get these nice changes into the emc.vpppg parallel before Lin starts his next EE2 review. Thanks!

**Review by PerryShafran-NOAA — APPROVED** [review]
Approved for merge.

**Pull request merged**